doc/source/release/1.15.0-notes.rst

```diff
@@ -0,0 +1,114 @@
+==========================
+SciPy 1.15.0 Release Notes
+==========================
+
+.. note:: SciPy 1.15.0 is not released yet!
+
+.. contents::
+
+SciPy 1.15.0 is the culmination of X months of hard work. It contains
+many new features, numerous bug-fixes, improved test coverage and better
+documentation. There have been a number of deprecations and API changes
+in this release, which are documented below. All users are encouraged to
+upgrade to this release, as there are a large number of bug-fixes and
+optimizations. Before upgrading, we recommend that users check that
+their own code does not use deprecated SciPy functionality (to do so,
+run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
+Our development attention will now shift to bug-fix releases on the
+1.15.x branch, and on adding new features on the main branch.
+
+This release requires Python 3.10+ and NumPy 1.23.5 or greater.
+
+For running on PyPy, PyPy3 6.0+ is required.
```

Comment:
I've looked at this number in wonder for a while. I _really_ doubt such an ancient PyPy is going to work for anything these days. It should be 7.3+, which is the current series (though even that's probably not sufficient for early patch versions, as that series is ~4.5 [years old](https://github.com/pypy/pypy/tags?after=release-pypy3.6-v7.3.1rc1)). And even then, PyPy support has been spotty enough that we've dropped it completely in conda-forge for 1.13.

Reply:
Ok, I revised the PR to purge that line out, we've just been carrying that line around for a while without much thought.

I'm skipping CI in the revised commit--it was already green here.